src/client/search.js

```diff
@@ -8,7 +8,7 @@ const resultsContainer = document.querySelector("#observablehq-search-results");
 const activeClass = "observablehq-link-active";
 let currentValue;
 
-const index = await fetch(import.meta.resolve(global.__minisearch))
+const index = await fetch(import.meta.resolve("observablehq:minisearch.json"))
```

Comment:
Now that we’ve implemented configurable import resolution in `rollupClient`, we don’t need to special-case the `minisearch.json` resolution (which was already content-hashed since it changes often); we can use the normal mechanism for import resolution instead.